src-backend/registration/tests/test_user.py

```diff
@@ -10,8 +10,5 @@ def setUp(self):
         self.test_user.save()
 
     def test_user_has_token(self):
-        try:
-            token = Token.objects.get(user=self.test_user)
-            self.assertFalse(token is None)
-        except ObjectDoesNotExist:
-            self.assertTrue(False)
+        token = Token.objects.get(user=self.test_user)
```

Comment:
Do exceptions cause the tests to fail in a functional sense, or does the test crash? I feel like it would be better to catch exceptions